### DISCUSSION TITLE
Uncaught (in promise)  TypeError: Failed to execute 'getValue' on 'CookieDeprecationLabel': Illegal invocation

### DISCUSSION THREAD

**Discussion by elizabthpazp** [johannschopplich/nuxt-gtag#87] [General]
In the chrome console gtag/js?id=... I have this error:
Uncaught (in promise) 
TypeError: Failed to execute 'getValue' on 'CookieDeprecationLabel': Illegal invocation

in gtag.js, the error occurs here:

try {
    tc.cookieDeprecationLabel ? (a("pending"),
    (0,
    tc.cookieDeprecationLabel.getValue)().then(a)) : a("noapi") // error here
} catch (b) {
    a("error")
}

"nuxt": "^3.12.4",
"nuxt-gtag": "^2.1.0",

**Comment by johannschopplich** [comment]
This is an issue of the official gtag.js library. This is not related to this module, since it solely injects the official Google Tag library.
